open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/janus-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/openlineageconnectors/janusconnector/utils/EdgeLabelsMainGraph.java

```diff
@@ -0,0 +1,13 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.openconnectors.governancedaemonconnectors.openlineageconnectors.janusconnector.utils;
+
+public enum EdgeLabelsMainGraph {
+    EDGE_LABEL_COLUMN_AND_PROCESS,
+    EDGE_LABEL_TABLE_AND_PROCESS,
+    EDGE_LABEL_SEMANTIC,
+    EDGE_LABEL_GLOSSARYTERM_TO_GLOSSARYTERM,
+    EDGE_LABEL_SUBPROCESS_TO_PROCESS,
+    EDGE_LABEL_CONDENSED,
+    EDGE_LABEL_INCLUDED_IN
```

Comment:
Already exists in GraphConstants but unless I am mistaken they are not being removed there?